spring-boot-project/spring-boot/src/main/java/org/springframework/boot/logging/logback/LogbackLoggingSystem.java

```diff
@@ -474,7 +473,7 @@ private void withLoggingSuppressed(Runnable action) {
 	/**
 	 * {@link LoggingSystemFactory} that returns {@link LogbackLoggingSystem} if possible.
 	 */
-	@Order(Ordered.LOWEST_PRECEDENCE)
```

Comment:
There are [several other places where we use `@Order(Ordered.LOWEST_PRECEDENCE)`. If we're going to change one, we should change them all. Personally, I don't think we should change any of them as I prefer to see the order explicitly declared.

Reply:
That makes sense for the sake of consistency, I removed it because IntelliJ usually doesn't recommend explicitly mentioning default values for annotations. Have added the constant back.